src/Module.ts

```diff
@@ -1015,6 +1024,9 @@ export default class Module {
 				// export * as name from './other'
 
 				const name = node.exported.name;
+				if (this.hasExportNamedAs(name)) {
+					this.error(logDuplicateExportError(name), node.exported.start);
```

Comment:
I would hope we add `function` tests for all uncovered error cases. Testing errors is pretty simple in `function` tests as you just add an object-valued `error` property to the test config, and the failing test should tell you which properties to add to the object.
Basically it ignores the `stack`, and for the frame there is a normalizer so that when you start the expected frame with back-tick -> line-break -> indentation, then line-break and indentation will be ignored and indentation is sliced from all subsequent lines before comparing.
But I can also support here.